schema.ts

```diff
@@ -1423,15 +1423,46 @@ export interface AnyonePrincipal {
 type Principal = UserPrincipal | GroupPrincipal | AnyonePrincipal | DomainPrincipal;
 
 /**
- * This represents the definition of a permission in the external system.
+ * The type of permission.
+ *
+ * TODO(drew): Unhide this
+ * @hidden
+ */
+export enum PermissionType {
+  Delegated = 'delegated',
```

Comment:
Can you include some basic JSDoc describing what the delegated and direct mean, when to use them, etc?